docs/concepts/fields.md

```diff
@@ -733,6 +772,25 @@ print(b.model_dump())
 #> {'width': 1.0, 'height': 2.0, 'depth': 3.0, 'volume': 6.0}
 ```
 
+As with regular fields, computed fields can be marked as being deprecated:
+
+```py
+from typing_extensions import deprecated
+
+from pydantic import BaseModel, computed_field
+
+
+class Box(BaseModel):
+    width: float
+    height: float
+    depth: float
+
+    @computed_field
+    @deprecated("'volume' is deprecated")
+    def volume(self) -> float:
+        return self.width * self.height * self.depth
```

Comment:
Haven't documented `@computed_field(deprecated=...)` as it is equivalent but explicitly using the `@deprecated` decorator will also include typing support